lib/Command/CrawlCommand.php

```diff
@@ -54,6 +55,7 @@ protected function configure()
         $this->addOption(self::OPT_OUTPUT, 'o', InputOption::VALUE_REQUIRED, 'Output file');
         $this->addOption(self::OPT_NO_DEDUPE, 'D', InputOption::VALUE_NONE, 'Do not de-duplicate URLs');
         $this->addOption(self::OPT_DESCENDANTS_ONLY, 'l', InputOption::VALUE_NONE, 'Only crawl descendants of the given path');
+        $this->addOption(self::OPT_FIRST_EXTERNAL_ONLY, 'f', InputOption::VALUE_NONE, 'Only crawl the first external not its child');
```

Comment:
I'm not sure if `-f` would be the best option here. Generally i'd expect `-f` to be `--force`.

Reply:
maye `x`? but thinking of just surveying existing tools (wget, curl, that python linkchecker) to find a best fit for these options